bazel/thirdparty/libpciaccess.BUILD

```diff
@@ -1,22 +1,71 @@
-load("@rules_foreign_cc//foreign_cc:defs.bzl", "configure_make")
-
-filegroup(
-    name = "srcs",
-    srcs = glob(["**"]),
-)
-
-configure_make(
+cc_library(
```

Comment:
When creating custom builds it is useful to document about how the build was created (porting the options in the autoconf build?) for future upgraders. Especially if it’s copied from somewhere (like maybe for hwloc in the future)

Reply:
fixed